pyhealth/datasets/eicu.py

```diff
@@ -414,8 +427,11 @@ def medication_unit(v_info):
             for offset, code in zip(v_info["drugstartoffset"], v_info["drugname"]):
                 # compute the absolute timestamp
                 timestamp = v_encounter_time + pd.Timedelta(minutes=offset)
+                
+                # reduce coding errors by removing whitespaces
+                medication_code = re.sub(r"\s+", '', code)
                 event = Event(
-                    code=code,
+                    code=medication_code,
```

Comment:
The treatment string is a character sequence (https://eicu-crd.mit.edu/eicutables/treatment/), so when pyhealth slaps it into the code field when parsing, it's not coming in as some standardized code, but something like one of these (these are 3 different treatment visits):
```
'neurologic|seizure therapy|anticonvulsant|gabapentin',
'cardiovascular|vascular disorders|anticoagulant administration|conventional heparin therapy|subcutaneous',
'pulmonary|medications|antibacterials|fourth generation cephalosporin|cefepime'
```
Here are 3 examples of code entries for medication:
```
'platelets x 1000',
'2 ML VIAL: FUROSEMIDE 10 MG/ML IJ SOLN',
'SPIRONOLACTONE 25 MG PO TABS'
```

Reply:
Future work: the composite codes shown should be split into their own distinct codes.